runner/src/server/services/cacheService.ts

```diff
@@ -148,7 +159,9 @@ export const catboxProvider = () => {
    */
   const provider = {
     constructor: redisHost ? CatboxRedis : CatboxMemory,
-    options: {},
+    options: {
+      maxByteSize: 1,
```

Comment:
What is this option doing? Why is it desirable to set?

Reply:
Thanks for flagging, this actually was for testing [CatboxMemory](https://hapi.dev/module/catbox-memory/api/?v=6.0.0) to see if we could simulate memory reaching capacity (to recreate the failure seen in demo) however this didn't seem to cause a hard limit. I've removed it as we're using redis instead.